### DISCUSSION TITLE
Trying to setup a 4.3.4 local server

### DISCUSSION THREAD

**Discussion by ayoub-khemissi** [The-Cataclysm-Preservation-Project/TrinityCore#391] [Q&A] [answered]
Hello,

Working on: Windows x64.

I've created a local 4.3.4 server and I'm looking for a client that can actually connect to it.

My bnetserver.exe and worldserver.exe are running well but it seems like the client I'm trying to use is not reaching my bnetserver.

What I've tryied so far:
- Patch my client with the connectionpatcher.exe, nothing changed, the patched exe doesn't reach the bnetserver on auth trial ;
- set realmlist 127.0.0.1 + set patchlist 127.0.0.1
- I've checked that the realmlist line is finely setted up.

Some screenshots:
bnetserver.exe & worldserver.exe running
![image](https://user-images.githubusercontent.com/4041328/218120752-4ac38fe9-6845-4d34-94f2-9b726d03ca31.png)

realmlist table
![image](https://user-images.githubusercontent.com/4041328/218120973-35a2a77a-150c-4a6f-8b87-141c740c7f82.png)

realmlist.wtf
![image](https://user-images.githubusercontent.com/4041328/218121152-da7af979-21cf-4dd3-83c1-6eb9e5321003.png)

Do you have any idea of what's wrong here? Also, some people on forums says that it's something with the x32 version that is not working, but when I'm trying to start WoW64.exe version (and the patched one) the game doesn't show up and I have to kill it in the taskmanager.

**Comment by funjoker** [comment]
realmlist.wtf is deprecated.

set realmlist is deprecated.

use the Config.WTF inside WTF folder.
There you need to use SET portal ""

**Reply by ayoub-khemissi** [reply]
Thank you for your answer, but just for you to know that even if it's deprecated (source btw?), the realmlist.wtf file is taken as the source for Config.wtf to set the realmlist and patchlist info. So if you modify the Config.wtf file it'll be overriden by the realmlist.wtf file content.

Also, I've already tryied to change the Config.wtf file just to give it a try without success.

Any other idea(s)?

**Reply by funjoker** [reply]
delete realmlist.wtf

**Reply by ayoub-khemissi** [reply]
Also tryied this, not working.

**Comment by ayoub-khemissi** [comment, marked as answer]
I've followed your tutorial step by step and it worked:

https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/wiki/Setup-and-troubleshooting-for-connecting

Here is what I did to make it work:

== SERVER SIDE ==

- Make sure the realmlist table has an entry with the correct address (in my case 127.0.0.1 like in my screen above).
- Make sure auth and worldserver are running well (like in my screen above).
- Create an account in the worldserver console ie: `bnetaccount create address@mail.com password`

== CLIENT SIDE ==

- Get the x64 client version of WoW 4.3.4 : http://us.media.battle.net.edgesuite.net/downloads/wow-installers/live/WoWLive-64-Win-15595.zip
- Add the WoW.mfil with the following content:
`version=2
server=akamai
    location=http://dist.blizzard.com.edgesuite.net/wow-pod-retail/EU/15050.direct/
manifest_partial=wow-15595-0C3502F50D17376754B9E9CB0109F4C5.mfil`
- Delete Data/{local ie: enUS}/realmlist.wtf file
- Modify the WTF/Config.wtf file to add the following line (same as the address field in the auth.realmlist table)  : 
`SET portal "127.0.0.1"`
- Patch the Wow-64.exe with the connection_patcher.exe, also move the original files (none "_Patched") to an OLD directory
- Start Wow-64.exe and log with the bnet account created in the server side steps (address@mail.com/password)

Thank you, this can be closed.
